pydm/widgets/shell_command.py

```diff
@@ -11,14 +11,26 @@
 from qtpy.QtGui import QCursor, QIcon, QMouseEvent, QColor
 from qtpy.QtCore import Property, QSize, Qt, QTimer
 from qtpy import QtDesigner
+from PyQt5.QtCore import Q_ENUMS
```

Comment:
Could probably just still pull this from `qtpy.QtCore` until the enum changes happen

Reply:
Good call